source/renderer/app/utils/hardwareWalletUtils.js

```diff
@@ -38,6 +39,11 @@ export const KEY_PREFIXES = {
 
 // Helpers
 
+const receiverAddressTypes: AddressType[] = [0, 1, 2, 3, 4, 5, 6, 7, 8];
```

Comment:
You could wrap this array in a `Set` to reduce time complexity `new Set([0, 1, 2, 3, 4, 5, 6, 7, 8])` in the function `isReceiverAddressType`

Reply:
You're right, updated :)